src/patternfly/components/Table/table-tree-view.scss

```diff
@@ -292,11 +292,8 @@ $pf-v6-c-tree-view--MaxDepth: 10;
 
     grid-row: 1;
     grid-column: 2;
-
-    > .#{$menu-toggle} {
-      margin-block-start: var(--#{$table}--m-tree-view-grid--c-menu-toggle--MarginBlockStart);
-      margin-block-end: var(--#{$table}--m-tree-view-grid--c-menu-toggle--MarginBlockEnd);
-    }
+    margin-block-start: var(--#{$table}--m-tree-view-grid__action--MarginBlockStart);
+    margin-block-end: var(--#{$table}--m-tree-view-grid__action--MarginBlockEnd);
```

Comment:
This moves the margin to the actions container since anything can go in the `__action` cell (not just a menu-toggle or button). @mattnolting do you see any problem with the margin on `__action`? It also has a bunch of padding and stuff but aligns fine as far as I can tell.